aikau/src/main/resources/alfresco/renderers/_ActionsMixin.js

```diff
@@ -192,19 +192,19 @@ define(["dojo/_base/declare",
          if (this.mergeActions === true)
          {
             // Add the actions on the currentItem and then add additional custom actions...
-            this.currentItem.actions && array.forEach(this.currentItem.actions, lang.hitch(this, this.addAction));
-            this.customActions && array.forEach(this.customActions, lang.hitch(this, this.addAction));
+            lang.isArray(this.currentItem.actions) && array.forEach(this.currentItem.actions, lang.hitch(this, this.addAction));
```

Comment:
Ok - I was trying to avoid adding another dependency to the module, but can certainly add ObjectTypeUtils.
As far as I am aware, all the type-checking that both lang and ObjectTypeUtils do can already be done in a native way supported across all (supported) browsers (considering IE 8 to be unsupported as it is not supported my MS anymore).